### PR TITLE
Active storage for car

### PR DESCRIPTION
**Pull request by TehanovAnton**

### CODE REVIEW

File: app/services/pexels_service.rb [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    @orientation = orientation
+  end
+
+  def photos
+    @client.photos.search(
+      description,
+      size: size,
+      orientation: orientation
+    )
+           .photos
```
Comment (blackp0is0n) [inline app/services/pexels_service.rb]: ?

File: app/views/cars/_form.html.erb [hunk 2]
```diff
@@ -11,6 +11,18 @@
     </div>
   <% end %>
 
+  <% photos.each do |photo| %>
+    <%= image_tag photo, size: '100x100' %>
```
Comment (blackp0is0n) [inline app/views/cars/_form.html.erb]: Image tag size should be limited by `CSS`

File: lib/tasks/pexel.rake [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'pexels'
+require './app/services/pexels_service.rb'
+
+namespace :pexel do
+  desc 'TODO'
```
Comment (blackp0is0n) [inline lib/tasks/pexel.rake]: Pls make description and name task according it's functionality. Currently I cannot understand what is this doing without checking it's code. Pls deal with it.

File: lib/tasks/pexel.rake [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+  task download: :environment do |t|
```
Comment (blackp0is0n) [inline lib/tasks/pexel.rake]: Same

File: lib/tasks/pexel.rake [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  task download: :environment do |t|  
+    pexels_service = PexelsService.new('bmw auto', size: :medium, orientation: :square)
+
+    3.times do |i|
+      source = pexels_service.photo_src(i)
+      pexels_service.upload(source)
+    end
+  end
+
+  task :random_attach, [:count] => :environment do |t, args|
```
Comment (blackp0is0n) [inline lib/tasks/pexel.rake]: Same

**Review by blackp0is0n — CHANGES_REQUESTED** [review]

File: app/views/cars/show.html.erb [hunk 6]
```diff
@@ -1,5 +1,17 @@
+<style>
```
Comment (blackp0is0n) [inline app/views/cars/show.html.erb]: Please do not use `<style>` tag, use css instead

File: app/views/cars/_form.html.erb (hunk 2 above)
Comment (blackp0is0n) [reply app/views/cars/_form.html.erb]: So if you moved size to css, I suppose we should drop `size` there

File: app/views/cars/_form.html.erb [hunk 7]
```diff
@@ -11,6 +11,18 @@
...
+    <%= radio_button_tag :photo, photo, false %>
+  <% end %>
+  <br>
+
+  <%= form.input :photo, as: :file %>
+
+  <% if @car_form.photo.attached? %>
+    <%= image_tag @car_form.photo.variant(resize_to_limit: [100, 100]), size: '100x100' %>
```
Comment (blackp0is0n) [inline app/views/cars/_form.html.erb]: The same about size

File: coverage/.last_run.json [hunk 8]
```diff
@@ -0,0 +1,5 @@
+{
+  "result": {
```
Comment (blackp0is0n) [inline coverage/.last_run.json]: Please add `coverage` folder to gitignore, I don't think it should be in repo

**Review by blackp0is0n — CHANGES_REQUESTED** [review]

File: app/views/cars/show.html.erb (hunk 6 above)
Comment (TehanovAnton) [reply app/views/cars/show.html.erb]: it's for a long time

**Review by blackp0is0n — APPROVED** [review]

**Pull request merged**